docs/sveltia-migration.md

```
@@ -0,0 +1,325 @@
+# Sveltia CMS 導入ガイド
+
+このサイト（Astro + Content Collections + Cloudflare Workers）に [Sveltia CMS](https://sveltiacms.app/) を導入し、
+**「ローカルエディタでも、ブラウザの管理画面でも書ける。画像は R2 にホストされる」** 体制を作るためのガイド。
+
+## 全体像
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)